Makefile

```
@@ -44,7 +44,7 @@ check-fmt: .venv ## Check format code with black
 .PHONY: check-typing
 check-typing: .venv ## Check typing with mypy
 	@echo "[CHK] typing"
-	poetry run mypy ocean_spark tests
+	poetry run mypy ocean_spark tests --ignore-missing-imports
```

```suggestion
	poetry run mypy ocean_spark tests
```

We can add `ocean_spark_connect` into the mypy.ini file

```toml
[mypy-ocean_spark_connect.*]
ignore_missing_imports = True
```

As an additional thing we need to add a `py.typed` marker file into the `ocean_spark_connect` module repo so that this just works.